Undertimes

Undertimes opened the pull request:
Le pnj et ses quêtes journalières sont prêtes! Les différentes quêtes fonctionnent (tests réussis sur le serveur dev) et le pnj se débloque pour les joueurs ayant atteint le grade d'écuyer.

Undertimes committed:
Quêtes journalières du Sanctuaire

Ajout de la première série de quêtes journalières du Sanctuaire. Pour le moment, uniquement les quêtes de difficulté 1, mais fonctionnelles.

Undertimes committed:
Ajout de nouvelles quètes

Nouvelles quètes de niveau 2, améliorations générales.

Undertimes committed:
Messages pour implantation

Modifications mineures pour rendre les missions fonctionnelles.

Undertimes committed:
Finalisation des quètes lvl.1 et lvl.2

Correction de bugs après tests sur le serveur dev. Le pnj et ses quètes sont prêtes à l'ajout sur le serveur live!

crashoz commented on BetonQuest/Undertimes/Quêtes Sanctuaire/main.yml:
```diff
@@ -0,0 +1,3 @@
+npcs:
```
Mettre des id de pnj recyclés sur le prod et les placer aux endroits adéquats

crashoz reviewed (changes requested):
C'est tout bon, je met juste en attente jusqu'à ce qu'on ait les pnj de placés sur le prod

HollishKid commented:
Ce PR m'a l'air très bien, mais je me pose quelques questions.
- n'y a-t-il pas un niveau de dossiers en trop ? UnderTimes > Quêtes sanctuaire
Il me semble que c'est théoriquement possible, je me pose plutôt des questions de "facilité" ;
- ne va-t-il pas y avoir de souci dans la définition des tags, objectifs etc. avec un espace et un caractère spécial dans un nom de package ?
On aurait du UnderTimes.Quêtes Sancutaire.nom_du_tag ; ce qui me parait pas optimal ?
- a plus long terme on n'aurait-t-on pas du mal à faire référence à ce package ?
- une fois en place on ne peut pas le modifier, puisque les joueur auront déjà des tags etc. du format que j'ai donné plus haut.

crashoz commented:
Effectivement je n'avais pas vu, ce serait bien mieux d'avoir les quêtes dans un seul dossier UnderTimes je suis d'accord.

Undertimes commented:
En effet, il serai plus efficace de ne pas avoir de sous dossier, sous tout les angles. 

Un autre détail que j'ai oublié de préciser est que le pnj 'joseph' situé dans le château royal peut être merge avec un autre pnj lié au sanctuaire, par exemple l'un des pnj du second étage, étant donné que ces quêtes ne sont accessibles qu'aux écuyers. Si vous avez une idée, n'hésitez pas. 

_Sent from my Sony D5803 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Undertimes committed:
Suppression dossier

Sous-dossier supprimé pour plus de clarté.

Undertimes commented:
Le sous-dossier est supprimé, y a-t-il autre chose à modifier?

HollishKid commented on BetonQuest/Undertimes/Quêtes Sanctuaire/conversations/gardien_sanctuaire.yml:
```diff
@@ -0,0 +1,102 @@
+    text: 'Mhhh... Vous pourriez accomplir quelques quêtes en but de nous aider. Vous seriez récompensé par une partie de notre sagesse.'
+    pointers: 'reponseaide2'
+  'missions4':
+    text: 'Bien. Quelle difficulté voulez-vous?'
```
Je vais faire le pointilleux sur la ponctuation, comme ça ce sera fait maintenant, plutôt que de corriger par PR séparé.

- Les ponctuations "doubles" ( : ; ! ?) sont toujours précédés d'un espace.
- Les ponctuations "simples" ( . , ...) ne le sont jamais.

HollishKid commented on BetonQuest/Undertimes/Quêtes Sanctuaire/conversations/gardien_sanctuaire.yml:
```diff
@@ -0,0 +1,102 @@
+    condition: 'tag_endquestlvl1'
+    event: 'questreward1'
+  'josephquest':
+    text: 'Apportez ce paquet au Chateau royal.'
```
Chateau => Château

HollishKid commented on BetonQuest/Undertimes/Quêtes Sanctuaire/items.yml:
```diff
@@ -0,0 +1,4 @@
+josephpackage: 'CHEST name:&2Colis_du_Sanctuaire lore:A_livrer_à_Joseph_au_Chateau!'
```
Peut être faudrait-il faire un quest item de cet objet ? (se range dans le /sac et ne peux jamais être perdu, sauf si repris par BQ)

Pour ce faire, il faudrait :
lore:A_livrer_à_Joseph_au_Château_!;§2Objet_de_Quête'

HollishKid commented on BetonQuest/Undertimes/Quêtes Sanctuaire/items.yml:
```diff
@@ -0,0 +1,4 @@
+josephpackage: 'CHEST name:&2Colis_du_Sanctuaire lore:A_livrer_à_Joseph_au_Chateau!'
+gold: 'GOLD_INGOT lore:none'
+iron: 'IRON_INGOT lore:none'
+questbone: 'BONE name:&2Os_mystique lore:Dans_votre_main,_il_en_émane_une_force_étrange.'
```
lore:Dans_votre_main;il_en_émane_une_force_étrange.

Un peu long sur une seule ligne de lore sinon, non ?

HollishKid commented on BetonQuest/Undertimes/conversations/gardien_sanctuaire.yml:
```diff
@@ -0,0 +1,102 @@
+    text: 'Mhhh... Vous pourriez accomplir quelques quêtes en but de nous aider. Vous seriez récompensé par une partie de notre sagesse.'
+    pointers: 'reponseaide2'
+  'missions4':
+    text: 'Bien. Quelle difficulté voulez-vous?'
```
Je vais faire le pointilleux sur la ponctuation.
Pour toute double ponctuation ( : ; ! ? ) il faut le faire précéder d'un espace.
(Pour ne pas spammer, je le relève à les première occurrence, mais y'en a quelques un dans tes conversations & events.)

HollishKid commented on BetonQuest/Undertimes/conversations/gardien_sanctuaire.yml:
```diff
@@ -0,0 +1,102 @@
+    condition: 'tag_endquestlvl1'
+    event: 'questreward1'
+  'josephquest':
+    text: 'Apportez ce paquet au Chateau royal.'
```
Château au lieu de Chateau

HollishKid commented on BetonQuest/Undertimes/items.yml:
```diff
@@ -0,0 +1,4 @@
+josephpackage: 'CHEST name:&2Colis_du_Sanctuaire lore:A_livrer_à_Joseph_au_Chateau!'
```
Peut être serait-ce une idée de faire un objet de quête ? (ne se perd jamais et peut être rangé dans le /sac)

Il faudrait alors :
josephpackage: 'CHEST name:&2Colis_du_Sanctuaire lore:A_livrer_à_Joseph_au_Château_!;§2Objet_de_Quête'

HollishKid commented on BetonQuest/Undertimes/items.yml:
```diff
@@ -0,0 +1,4 @@
+josephpackage: 'CHEST name:&2Colis_du_Sanctuaire lore:A_livrer_à_Joseph_au_Chateau!'
+gold: 'GOLD_INGOT lore:none'
+iron: 'IRON_INGOT lore:none'
+questbone: 'BONE name:&2Os_mystique lore:Dans_votre_main,_il_en_émane_une_force_étrange.'
```
La seule ligne de lore semble un peu chargée. Et si tu divisais en deux lignes ?

questbone: 'BONE name:&2Os_mystique lore:Dans_votre_main;il_en_émane_une_force_étrange.'

Undertimes committed:
Corrections mineures

Corrections en grande partie orthographiques dans les textes.

Undertimes commented:
Modifications appliquées, ça devrait être bon pour ce que Hollish a relevé.

HollishKid committed:
Espacement ponctuation "double"

HollishKid committed:
Ponctuation suite.

HollishKid committed:
Folder delay converti en objectif (1/2)

HollishKid committed:
Folder delay en objectif (2/2)

Undertimes committed:
Passage Citizen vers MM 1/2

Première partie du passage de Citizen vers MythicMobs. Fichier spawners à venir.

Undertimes committed:
Passage Citizen vers MM 2/2

Ajout des fichiers spawners.

Undertimes committed:
Merge branch 'master' into Undertimes

HollishKid commented on MythicMobs/Spawners/pnj_gardien.yml:
```diff
@@ -0,0 +1,25 @@
+  WarmupTimer: 0
+  CheckForPlayers: true
+  ActivationRange: 40
+  LeashRange: 50
```
Tu devrais passer le LeashRange à 1 pour que le PNJ reste bien là ou tu le fais spawner.

HollishKid commented on MythicMobs/Spawners/pnj_joseph.yml:
```diff
@@ -0,0 +1,25 @@
+  WarmupTimer: 0
+  CheckForPlayers: true
+  ActivationRange: 40
+  LeashRange: 50
```
Idem ici, LeashRange: 1

Undertimes replied:
Ah oui, je pensais l'avoir modifié pourtant... Bah, je m'en occupe tout de suite!

Undertimes committed:
Fix spawners

Modification de la leash range des spawners à pnj.

Undertimes committed:
Finitions passage MM

Finitions avant le merge.

Undertimes commented:
Plus que l'approbation de @HollishKid et je merge.
